src/lib.rs

```diff
@@ -162,6 +162,14 @@ impl ZWaveManager {
     pub fn get_state(&self) -> MutexGuard<State> {
         self.watcher.get_state()
     }
+
+    pub fn write_configs(&self) {
+        let state = self.get_state();
+        let controllers = state.get_controllers();
+        for (controller, _) in controllers {
+            controller.write_config();
+        }
+    }
```

Comment:
I think that this is fine for now. We should probably have stateful keep track of whether a write is needed and only write if a write is actually needed.